src/style.css

```diff
@@ -1 +1,90 @@
-/* Styles go here */
+/* add fonts */
+@font-face {
+  font-family: "Roboto";
+  src: url(./fonts/Roboto/Roboto-Medium.ttf);
+  font-weight: 500;
+}
+
+/* main bloks */
+.html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+}
+
+.body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+/* logo container */
+.logo__container {
+  display: flex;
+}
+
+.logo__link {
+  display: flex;
+
+  padding: 10px 0 10px 50px;
```

Comment:
Will be better to specify correct right and left paddings for the whole header (they're the same, 50px) instead of adding padding to the logo and to the navigation block